### ISSUE TITLE
BUG: jormungandr constantly starting battles stuck in station.

### ISSUE THREAD

**Issue by SuplexCity25284** [PavelZinchenko/event-horizon-frontier-builds#357]
As title states.  My merc controlled Jormungandr is starting out every battle stuck about 80% in the base.  It eventually works it's way out of this, but not for several minutes.  Once it does so, it sits around, fights, but flies EXTREMELY slowly, if it moves at all.  However if I take my ship and ram it at the head, it quickly escapes this position and then flies off normally.

**Comment by xdeadstokex** [comment]
it look like your pet snake is a little shy, aye, it need your encouragement to move faster after all.

**Comment by SuplexCity25284** [comment]
Lol

All jokes aside, it's a pain to have to worry about something im not controlling when, about 3 of every four battles it's bugged out like this.  It even happens in Arena.

**Comment by xdeadstokex** [comment]
Yes, long ships are weird, and it is frustrating when they stuck when you need them the most (my QoL post 4 post below can fix this problem, just by display where your ship will be before battle)

**Comment by SuplexCity25284** [comment]
That would be handy, for sure.

**Comment by SuplexCity25284** [comment]
It still occurs now, just not quite as often.  But I notice now that Jormy likes to just stop flying and sit, until a ship bumps into it.  

Has anyone else noticed this happen?
